### ISSUE TITLE
Keras API reproducibility

### ISSUE THREAD

**Issue by snassimr** [tensorflow/tensorflow#11585]
If any new option was added to ensure a reproducible results by setting some seed parameter
?
In Keras the issue was discussed [here](https://github.com/fchollet/keras/issues/2743)
According the issue the instability in results is due to weights random initialization.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
